File: Service_eHospital_DongNai_A/Implement/DM_DVT_ThoiGianService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class DM_DVT_ThoiGianService : BaseService<DM_DVT_ThoiGian, IDM_DVT_ThoiGianRepository>
    , IDM_DVT_ThoiGianService
    {
    private readonly IDM_DVT_ThoiGianRepository _DM_DVT_ThoiGianRepository;
    public DM_DVT_ThoiGianService(IDM_DVT_ThoiGianRepository DM_DVT_ThoiGianRepository) : base(DM_DVT_ThoiGianRepository)
    {
    _DM_DVT_ThoiGianRepository = DM_DVT_ThoiGianRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/DM_DVT_ThoiGian.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DM_DVT_ThoiGian } from './DM_DVT_ThoiGian.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DM_DVT_ThoiGianService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'DVT_ThoiGian_Id', 'MaDonViTinh', 'TenDonViTinh'];

    List: DM_DVT_ThoiGian[] | undefined;
    ListFilter: DM_DVT_ThoiGian[] | undefined;
    FormData!: DM_DVT_ThoiGian;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DM_DVT_ThoiGian";
    }
}

